core/src/test/java/org/elasticsearch/action/delete/DeleteResponseTests.java

```diff
@@ -92,8 +93,8 @@ private static DeleteResponse randomDeleteResponse() {
         ShardId shardId = new ShardId(randomAsciiOfLength(5), randomAsciiOfLength(5), randomIntBetween(0, 5));
         String type = randomAsciiOfLength(5);
         String id = randomAsciiOfLength(5);
-        long seqNo = randomIntBetween(-2, 5);
-        long version = (long) randomIntBetween(0, 5);
+        long seqNo = randomBoolean() ? randomNonNegativeLong() : randomIntBetween(0, 10000);
```

Comment:
Can we use 
`randomFrom(SequenceNumbersService.UNASSIGNED_SEQ_NO, randomNonNegativeLong(), randomIntBetween(0, 10000)) `

instead?